.github/ISSUE_TEMPLATE/bug-report.yaml

```
@@ -0,0 +1,51 @@
+name: Bug report 🐞
+description: File a bug report
+title: "[Bug] "
+labels: bug
+body:
+- type: checkboxes
+  id: existing-issue
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    options:
+    - label: I have searched the existing issues
+      required: true
```

Maybe also ask people to check the CHANGELOG to see if it is in a closed issue/the next release
```suggestion
- type: checkboxes
  id: existing-issue
  attributes:
    label: Is there an existing issue for this?
    description: Please search to see if an issue already exists for the bug you encountered.
    options:
    - label: I have searched the existing issues
      required: true
- type: checkboxes
  id: existing-issue
  attributes:
    label: Is this issue mentioned in the CHANGELOG?
    description: Please search to see if the bug you encountered is already addressed in the upcoming release.
    options:
    - label: I have searched the CHANGELOG and expect the bug to be reproducible on the devel branch
      required: true
```